environment/environment_templates.go

```diff
@@ -351,31 +393,38 @@ func getBasicDependencyGroup() *K8sManifestGroup {
 
 // addNetworkManifestToDependencyGroup adds the correct network to the dependency group and returns
 // an array of all groups, this should be called as the last function when creating deploys
-func addNetworkManifestToDependencyGroup(envName string, dependencyGroup *K8sManifestGroup, chainlinkGroup *K8sManifestGroup) K8sEnvSpecInit {
+func addNetworkManifestToDependencyGroup(envName string, chainlinkGroup *K8sManifestGroup, dependencyGroups []*K8sManifestGroup) K8sEnvSpecInit {
 	return func(config *config.NetworkConfig) (string, K8sEnvSpecs) {
+		var specs K8sEnvSpecs
+		indexOfLastElementInDependencyGroups := len(dependencyGroups) - 1
 		switch config.Name {
 		case "Ethereum Geth reorg":
-			dependencyGroup.manifests = append(
-				dependencyGroup.manifests,
+			dependencyGroups[indexOfLastElementInDependencyGroups].manifests = append(
```

Comment:
You could probably make a reference to `dependencyGroups[indexOfLastElementInDependencyGroups].manifests` above the switch to use for all of these to make it more readable. Not a big deal though.